app/views/refinery/authentication/devise/admin/users/_user.html.erb

```diff
@@ -1,3 +1,13 @@
+<% # setup params for various action links
+  edit_url =  refinery.edit_authentication_devise_admin_user_path(user)
+  delete_url = refinery.authentication_devise_admin_user_path(user)
+  delete_options = {
```

Comment:
If we use `||=` instead of `=` then it means that these can be passed in as variables using local assigns in Rails, so that they can be overridden when using this partial.

Reply:
Done